tmt/base.py

```diff
@@ -1093,6 +1106,7 @@ class Test(
         'order',
         'result',
         'check',
+        'restart_test_on_exit_code',
```

Comment:
Shouldn't we show all three keys?

Reply:
Fixed in b3faad7c79334881fedd6a3a35f0323f698e080d.